bob-apps/bin/bobp.rs

```diff
@@ -550,8 +550,11 @@ fn print_periodic_stat(
     let print_put = (behavior_flags & PUT_FLAG) > 0;
     let print_get = (behavior_flags & GET_FLAG) > 0;
     let print_exist = (behavior_flags & EXIST_FLAG) > 0;
+    let multiline = (u8::from(print_exist) + u8::from(print_put) + u8::from(print_get)) > 1;
     while !stop_token.load(Ordering::Relaxed) {
         thread::sleep(pause);
+        let elapsed = start.elapsed().as_millis() as f64 / 1000.;
```

Comment:
`as_secs()` is enough. Fraction of seconds is not needed, because pause interval is 1 second